tools/check-symbol-names

```diff
@@ -20,11 +20,19 @@ set -o pipefail
 
 nm -A -P "$@" | awk '
 # ignore caml_foo, camlFoo_bar, _caml_foo, _camlFoo_bar
-$2 ~ /^(_?caml[_A-Z])/ { next }
+$2 ~ /^(_?[c|C]aml[_A-Z])/ { next }
 # ignore local and undefined symbols
-$3 ~ /^[rbdtsU]$/ { next }
+$3 ~ /^[a-zU]$/ { next }
 # ignore "main", which should be externally linked
 $2 ~ /^_?main$/ { next }
+$2 ~ /^_?wmain$/ { next }
+# for x86 PIC mode
+$2 ~ /^___x86.get_pc_thunk./ { next }
+# for mingw32
+$2 ~ /^.debug_/ { next }
+#window unicode support
+$2 ~ /^_win_multi_byte_to_wide_char$/ { next }
+$2 ~ /^_win_wide_char_to_multi_byte$/ { next }
```

Comment:
I can't find any reason why these two exported functions are not prefixed with `caml_`.  @dra27: any opinion?

Reply:
I think it's a hangover from the original patch, which used `win_` for its C stubs. There's no reason not to change this in another PR, as you say. The two functions are exported inside `CAML_INTERNALS` and there's no reason to use them directly unless you're writing stuff which is explicitly for Windows (you want `caml_copy_string_of_os` and `caml_stat_strdup_to_os` normally, as documented in the manual) - a search on GitHub suggests they're not being used anywhere.